lib/utils.js

```diff
@@ -215,3 +218,16 @@ export const flattenObjectDeep = obj =>
   );
 
 export const stripHTML = htmlContent => sanitizeHtml(htmlContent, { allowedTags: [], allowedAttributes: {} });
+
+export const checkRequiresAddress = values => {
```

Comment:
Also, since this is in `checkRequiresAddress`, there's not much clue that we're dealing with an expense here. I would personally move this to `components/expenses/lib/utils.js`, an alternative is to rename it to something like `checkExpenseRequiresAddress`.

Reply:
Moved the function to `components/expenses/lib/utils.js`. 👍🏽